sdcm/sct_runner.py

```diff
@@ -306,9 +345,20 @@ def create_instance(self, test_id: str, test_duration: int, instance_type: str =
             test_duration=test_duration,
         )
 
+    @classmethod
+    @abstractmethod
+    def list_sct_runners(cls) -> list[SctRunnerInfo]:
+        ...
+
+    @staticmethod
+    @abstractmethod
+    def terminate_sct_runner_instance(sct_runner_info: SctRunnerInfo) -> None:
+        ...
+
 
 class AwsSctRunner(SctRunner):
     """Provision and configure the SCT Runner on AWS."""
+    CLOUD_NAME = "aws"
```

Comment:
We already have "cloud_provider" attribute. Wouldn't it be enough?

Reply:
This is a great finding: actually, I found that I need to have cloud name as a class attribute (this is the reason why I introduced `CLOUD_NAME`). So, I'll remove old instance attribute named `cloud_provider' and replace it with `CLOUD_PROVIDER` on the class level. Thank you.